src/mumble/MainWindow.cpp

```diff
@@ -505,12 +505,13 @@ bool MainWindow::nativeEvent(const QByteArray &, void *message, long *) {
 
 void MainWindow::closeEvent(QCloseEvent *e) {
 	ServerHandlerPtr sh = Global::get().sh;
+	QuitBehavior quitBehavior = Global::get().s.quitBehavior;
 
+	if (!bSuppressAskOnQuit && ((quitBehavior == QuitBehavior::ALWAYS_ASK)
```

Comment:
In the light of the new changes, I think we should rename `bSuppressAskOnQuit` to something like `quitBehaviorOverride` or `forceQuit` or something like this :thinking: